CondCore/ESSources/test/stubs/TestConcurrentIOVsCondCore.cc

```diff
@@ -174,8 +174,9 @@ namespace edmtest {
     // The original reference file was created using a version
     // of the code before concurrent IOVs were implemented.
     for (auto const& object : testObjects_) {
-      std::cout << "TestConcurrentIOVsCondCore: lumi = " << object.lumi_ << " position: (" << object.x_ << ", "
-                << object.y_ << ", " << object.z_ << ")  iov = " << object.start_ << ":" << object.end_ << std::endl;
+      edm::LogPrint("TestConcurrentIOVsCondCore")
```

Comment:
apparently the change in formatting of this is responsible of the failure in the unit test `UnitTestConcurrentIOVsCondCore` ([link]( https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-d5d596/21126/unitTests/src/CondCore/ESSources/test/UnitTestConcurrentIOVsCondCore/testing.log))

Reply:
Fixed in 2159f8d.
The unitTest was redirecting the cmsRun output to a log file and then grepping on that to compare the output with what is stored in `unit_test_outputs/TestConcurrentIOVsCondCore.log`.
Now I have changed the unitTest so that:
 - the cmsRun output is handled using the `MessageLogger`
 - the grepping now works on the `MessageLogger` output

Running locally `scramv1 b runtests` now leads to
`---> test UnitTestConcurrentIOVsCondCore succeeded`